file/rotator.go

```diff
@@ -380,7 +384,12 @@ func (r *Rotator) closeFile() error {
 	}
 	err := r.file.Close()
 	r.file = nil
-	return errors.Wrap(err, "failed to close active file")
```

Comment:
`Wrap` returns `nil` if the `err == nil`, `fmt.Errorf` does not, so had to adjust.